form.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Make sure anything that isn't ready for code review goes into its own branch for merging later. It's very difficult to unpick code like this from a merge so avoid the issue by using branches for any code you're working on rather than keeping everything in your story branch!